subsys/bluetooth/shell/bt.c

```diff
@@ -1069,13 +1069,14 @@ static void auth_pairing_complete(struct bt_conn *conn, bool bonded)
 		    addr);
 }
 
-static void auth_pairing_failed(struct bt_conn *conn)
+static void auth_pairing_failed(struct bt_conn *conn, u8_t reason)
 {
 	char addr[BT_ADDR_LE_STR_LEN];
 
 	bt_addr_le_to_str(bt_conn_get_dst(conn), addr, sizeof(addr));
 
-	shell_print(ctx_shell, "Pairing failed with %s", addr);
+	shell_print(ctx_shell, "Pairing failed with %s reason 0x%02x", addr,
+		reason);
```

Comment:
Is the alignment of this split line correct? At least my browser renders it like `reason` doesn't match the ( on the previous line.

Reply:
Was not aligned, fixed.